### DISCUSSION TITLE
Step by Step guide for analyzing rat vocals

### DISCUSSION THREAD

**Discussion by pepipinkas** [DrCoffey/DeepSqueak#195] [Q&A]
Dear all,
Sorry for asking but I did not found a step by step guide on how to use deepSqueak allthough it should be self explaining. But as I am not working in science just only in IT and need to prepare deepSqueak for one of our users. But we both did not found out at the moment how the right way is.
We have selected Audio and Output folder
we then choose a file from the dropdown and load it, then we detect the calls and load those but we do not have any one marked unless we mark those calls and Accept them and save the session but for a new wav file we have to do the same again with drawing and accepting how can it be given to learn so we have to go only through one file and according to that one it will be detected in the other files?
Thanks a lot and appreciate any help

**Comment by DrCoffey** [comment]
Hey @pepipinkas. The wiki has most of the basic information for getting things up and running.

https://github.com/DrCoffey/DeepSqueak/wiki
